### PR TITLE
Resource: jenkins_credential_ssh

### PR DESCRIPTION
**Pull request by jaceq**
# What Is Changing
Added new resource: credential_ssh allowing to store SSH key in jenkins.

# Test Steps

```sh
make test
make testacc
```

### CODE REVIEW

**Comment by jaceq** [comment]
@taiidani any update here ? ;)

**Comment by taiidani** [comment]
@jaceq Sorry about this! New job means more distractions. I promise I'll review this weekend.

File: example/credentials_ssh.tf [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  username = "example-username"
+  privatekey = file(/some/path/id_rsa)
+}
+
+resource "jenkins_credential_ssh" "folder" {
+  name       = "some-id"
+  folder     = jenkins_folder.example.id
+  username   = "example-username"
+  privatekey = file(/some/path/id_rsa)
+}
```
Comment (taiidani) [inline example/credentials_ssh.tf:12]: Obviously this is a placeholder, but still probably good to have it be valid syntax :)

```
Error: Invalid expression
│ 
│   on credentials_ssh.tf line 4, in resource "jenkins_credential_ssh" "global":
│    4:   privatekey = file(/some/path/id_rsa)
│ 
│ Expected the start of an expression, but found an invalid expression token.
```

```suggestion
resource "jenkins_credential_ssh" "global" {
  name     = "some-id"
  username = "example-username"
  privatekey = file("/some/path/id_rsa")
}

resource "jenkins_credential_ssh" "folder" {
  name       = "some-id"
  folder     = jenkins_folder.example.id
  username   = "example-username"
  privatekey = file("/some/path/id_rsa")
}
```

File: docs/resources/credential_ssh.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+~> The "passphrase" and "privatekey" properties may leave plain-text values in your state file. Ensure that your state file is properly secured and encrypted at rest.
+
+## Example Usage
+
+```hcl
+resource "jenkins_credential_ssh" "example" {
+  name       = "example-id"
+  username   = "example-username"
+  privatekey = file(/some/path/id_rsa)
```
Comment (taiidani) [inline docs/resources/credential_ssh.md:13]:
```suggestion
  privatekey = file("/some/path/id_rsa")
```

**Review by taiidani — COMMENTED** [review]
High quality as usual @jaceq! I caught one thing around the privatekey paths, but otherwise LGTM

**Comment by jaceq** [comment]
Thank you for making that correction (and sorry for missing this).

**Pull request merged**